source/manual/data-gov-uk-common-tasks.html.md

```diff
@@ -1,240 +0,0 @@
-
-1. Create a release on [Publish Data][publish] or [Find Data][find] via GitHub. Choose a tag in the format `v[number].[number].[number]` (e.g. `v1.2.1`). Note there is no dot between the v and first number. Make sure you use [semantic versioning](https://semver.org/) for tags.
-2. Add decent notes covering what is being deployed.
-3. Publish the release. This remote triggers a deployment to production by running the `deploy.sh` script. This is defined in the Travis configuration file in each repo.
-4. Monitor deployment in Travis.
-5. In some cases, such as when migrations have been added, the data needs to be reimported and reindexed. See the [Importing data section](#import-data-from-legacy).
-6. If the data has been reindexed, [Find Data][find] probably needs to be deployed. Follow the same process by creating a git tag on [Find Data][find].
-
-> More documentation is available at [Using Travis CI to deploy to Cloud Foundry](http://cruft.io/posts/using-travis-ci-to-deploy-to-cloud-foundry/).
-
-## Deploy manually
-
-> These instructions can be used both on staging and production.
-
-### Applications (Find Data, Publish Data, Publish Data Worker)
-
-If you have to deploy apps manually on [PaaS][paas] (e.g. if Travis or GitHub are down):
-
-1. Use `scripts/deploy-production.sh` or `scripts/deploy-staging.sh` in the respective app repos.
-2. Make sure you deploy both Publish Data and Publish Data Worker at the same time.
-
-### Postgres
-
-[PaaS][paas] doesn’t let users drop and recreate databases in the Postgres service. Instead, if you’d like to rebuild your database you need to use cloudfoundry commands to:
-
-1. Unbind the old Postgres service from the apps using it (Publish Data and Publish Data Worker).
-2. Delete the service.
-3. Recreate the service.
-4. Rebind the service to the apps and restage them.
-5. Repopulate it from a [PaaS backup][paas-backup] or from a datadump file via Publish Data (see Syncing below)
-
-### Elasticsearch
-
-You shouldn’t normally need to re-instantiate an Elasticsearch service. [PaaS][paas] are responsible for keeping services working. However you can always do it by using cloudfoundry to:
-
-1. Unbind the old Elasticsearch service from the apps that use it.
-2. Delete the service.
-3. Recreate the service.
-4. Rebind the service to the apps and restage them.
-5. Populate it with data from a [PaaS backup][paas-backup] or from the [Publish Data][publish] database via the `search:reindex` task in the repo.
-
-Once set-up on [PaaS][paas] and bound to [Publish Data][publish] and [Find Data][find] everything should work. Environment variables should be provided automatically to the apps.
-
-To repopulate the index with dataset metadata:
-
-* From a PaaS backup: ask [PaaS support](/manual/data-gov-uk-incidents#data-loss).
-* From a Legacy dataset dump: use the `search:reindex` task in [Publish Data][publish]. You can specify a batch size as an argument `search:reindex[100]`, the default is 50.
-
-
-### Redis
```

Comment:
Should we keep this, as it may not be obvious to the people of the future that Redis is hosted elsewhere?

Reply:
So Redis is now on PaaS - did that yesterday :-)

Reply:
Lovely 👍